Framework/Source/Graphics/RenderGraph/RenderPassesLibrary.h

```diff
@@ -26,20 +26,23 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ***************************************************************************/
 #pragma once
+#include "RenderPassReflection.h"
 
 namespace Falcor
 {
     class RenderPass;
-    class RenderPassSerializer;
-
+    
     class RenderPassLibrary
     {
     public:
         using CreateFunc = std::function<std::shared_ptr<RenderPass>(const RenderPassSerializer&)>;
-        static void addRenderPassClass(const char* className, const char* desc, CreateFunc func);
+        static void addRenderPassClass(const char* className, const char* desc, CreateFunc func, RenderPassSerializer serializer = {});
         static std::shared_ptr<RenderPass> createRenderPass(const char* className);
+        static std::shared_ptr<RenderPass> createRenderPass(const char* className, const RenderPassSerializer& serializer);
```

Comment:
No need for a new function, changed the previous create to accept a serializer with a default argument of `{}`